neuralop/mpu/comm.py

```diff
@@ -96,72 +95,36 @@ def get_model_parallel_group():
     assert dist.is_initialized(), "Error, initialize torch.distributed first"
     return _MODEL_PARALLEL_GROUP  
 
-# get 
-def init(config, verbose = False):
-    
-    # set up global and local communicator
-    if config.distributed == "env":
-
-        world_size = int(os.getenv('WORLD_SIZE', 1))
-        world_rank = int(os.getenv('WORLD_RANK', 0))
-        port = int(os.getenv('MASTER_PORT', 0))
-        master_address = os.getenv('MASTER_ADDRESS')
-    
-    
-    elif config.distributed.wireup_info == "mpi":
 
-        import socket
-        from mpi4py import MPI
 
-        mpi_comm = MPI.COMM_WORLD.Dup()
-        world_size = mpi_comm.Get_size()
-        world_rank = mpi_comm.Get_rank()
-        my_host = '127.0.0.1'
-        port = 29500
-        master_address = mpi_comm.bcast(my_host, root=0)
-        os.environ["MASTER_ADDRESS"] = master_address
-        os.environ["MASTER_PORT"] = str(port)
-
-    else:
-        raise ValueError(f"Error, wireup-info {config.distributed.wireup_info} not supported")
+# get 
+def init(model_parallel_size:  int=1,
```

Comment:
Can we make the signature one line here?

Reply:
Done